Assignment-5/app/src/components/index.js

```diff
@@ -0,0 +1,121 @@
+
```

Comment:
Please fix this file same as you have done in assignment 4

Reply:
> Please fix this file same as you have done in assignment
 updated